src/main/java/TaskList.java

```diff
@@ -0,0 +1,33 @@
+import java.util.LinkedList;
+
+public class TaskList {
```

Comment:
I feel like this is a pretty useless abstraction... but I suppose it was asked for.

Reply:
Yeah lol no choice.. quite redundant I feel